packages/swarm-docs/src/components/nav.js

```diff
@@ -1,23 +1,53 @@
 // @flow
 import * as React from 'react';
 import { Link } from 'gatsby';
+type FileGroup = {
+    nodes: Array<File>
+};
+
+type File = {
+    relativeDirectory: string,
+    name: string
+}
 
 type Props = {
-    fileNames: Array<string>
+    files: Array<FileGroup>,
 };
 
+const transformFiles = (files: Array<FileGroup>) =>
+files.map(file =>
+  file.nodes.reduce((folderMap, node) => {
```

Comment:
looks like Prettier didn't convert the indentation to tabs here, and this function still isn't documented very well in terms of explaining what its purpose/expected behavior is